es6.md - german translation

fibric opened the pull request.

Commit:
german translation

Commit:
german translation, continue

Commit:
german translation, finalization

Commit:
Merge pull request #1 from fibric/ES6-german-1

ES6 german

Commit:
Update es6.md

Commit:
Merge pull request #2 from fibric/fibric-patch-1

Update es6.md

Commit:
Update es6.md

Commit:
Merge pull request #3 from fibric/fibric-patch-1

Update es6.md

Commit:
Update es6.md

Commit:
Merge pull request #4 from fibric/fibric-patch-1

Update es6.md

Commit:
Update es6.md

Commit:
Merge pull request #5 from fibric/fibric-patch-1

Update es6.md

Commit:
Update es6.md

Commit:
Merge pull request #6 from fibric/fibric-patch-1

Update es6.md

Commit:
Update es6.md

Commit:
Merge pull request #7 from fibric/fibric-patch-1

Update es6.md

Commit:
Update es6.md

Commit:
Merge pull request #8 from fibric/fibric-patch-1

Update es6.md

Commit:
Update es6.md

Commit:
Merge pull request #9 from fibric/fibric-patch-1

Update es6.md

Commit:
Update es6.md

Commit:
Merge pull request #10 from fibric/fibric-patch-1

Update es6.md

Commit:
Update es6.md

Commit:
Merge pull request #11 from fibric/fibric-patch-1

Update es6.md

Commit:
Update es6.md

Commit:
Merge pull request #12 from fibric/fibric-patch-1

Update es6.md

Commit:
change text of link

fhemberger commented on content/de/es6.md:
```diff
@@ -1,20 +1,20 @@
-*   Finally, all **in progress** features can be activated individually by their respective harmony flag (e.g. `--harmony_arrow_functions`), although this is highly discouraged unless for testing purposes.
+*   Alle als **shipping** bezeichneten Funktionen (jene die im V8-Projekt als stabil betrachtet werden), also solche wie [generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*), [Templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings), [neue String-Methoden](https://developer.mozilla.org/en-US/docs/Web/JavaScript/ ...
+*   Dann die **staged**-Funktionen. Diese mehr oder weniger vollständig Funktionen wurden noch nicht komplett getestet oder sind noch nicht auf dem aktuellen Stand der Spezifikationen. Diese werden vom V8 Team deswegen auch als instabil gehandelt (z.B. Verhaltensvorhersagen zu Ausnahmefällen sind noch nicht möglich). Vergleichbar war oder ist die Situation der [generators](https://developer.mozil ...
+*   Abschließend, alle **in progress**-Funktionen die nur einzelln mittels harmony Startparameter aktivierbar sind (z.B. `--harmony_arrow_functions`), davon ist aber dringend abzuraten, auser zu testzwecken.
```
einzeln

fhemberger commented on content/de/es6.md:
```diff
@@ -47,22 +49,22 @@ With io.js@1.x (V8 4.1+), all that complexity goes away. All harmony features ar
 
-*   [Classes](https://github.com/lukehoban/es6features#classes) (strict mode only)
-*   [Object literal extensions](https://github.com/lukehoban/es6features#enhanced-object-literals)
+*   [Classes](https://github.com/lukehoban/es6features#classes) (Nur im `'use strict';` Modus)
```
Ich würde eher "`strict`-Modus" schreiben.

snostorm commented:
Thanks @fibric. This seems to have a merge conflict. Once we get that out of the way and a thumbs up from another native speaker, we can proceed. /cc @iojs/iojs-de

fibric commented:
OK I think the merge conflict is caused by being 77 commits behind the master. I'll check later in hotel what I can do.

snostorm commented:
Any news or should we close this until ready?

fibric commented:
@snostorm 
reasoning about merge conflicts.
i've forked iojs v1.2 and on same date 1.3 was released. by creating this pr, i was already 100+ commits behind.
in fact, only es6.md file has changed. and has all comments and suggested edits inside.

snostorm commented:
If you update the PR to use the newer "version number" variable placeholders that would be great. https://raw.githubusercontent.com/iojs/website/master/content/en/es6.md

If you don't have time perhaps somebody in @iojs/iojs-de would have a few minutes to resolve/replace the PR?

ChristianGaertner commented on content/de/es6.md:
```diff
@@ -47,22 +49,22 @@ With io.js@1.x (V8 4.1+), all that complexity goes away. All harmony features ar
+## Wie erkenne ich, mit welcher V8 Version io.js ausgeliefert wurde?
 
-io.js provides a simple way to list all dependencies and respective versions that ship with a specific binary through the `process` global object. In case of the V8 engine, type the following in your terminal to retrieve its version:
+io.js bietet eine einfache Möglichkeit, alle Abhängigkeiten der jeweilig installierten Version, durch das globale Objekt `process` aufzulisten. Die V8-Engine stellt einen Befehl zur Verfügung, der die Versions-Informationen durch folgenden Konsolenbefehl ausgibt:
```
Der letzte Satz klingt komisch durch die Dopplung des Wortes "Befehl"; ich würde da dichter beim englischen Vorbild bleiben und nur "Durch folgenden Konsolen-Befehl kann die Version des V8-Engines ausgegeben werden." verwenden.

fibric replied:
Danke, das klingt gleich viel besser.

Commit:
reduce context duplication

Commit:
Merge pull request #13 from fibric/fibric-patch-1

reduce context duplication

Commit:
exchange io.js version by template variables

Commit:
Merge pull request #14 from fibric/fibric-patch-1

exchange io.js version by template variables

Commit:
remove orphaned paragraph

Commit:
Merge pull request #15 from fibric/fibric-patch-1

remove orphaned paragraph

nerdalytics committed:
Revert "Merge pull request #14 from fibric/fibric-patch-1"

This reverts commit df689ee41ca4a38536112688cfc586f64260fe8c, reversing
changes made to 9b20eb552b55b2bf6b7c54d5aadda37c209c08cd.

nerdalytics committed:
undo revert to revision

removed a paragraph by mistake. wanted to revert but identified wrong revision.

Commit:
new paragraph

Commit:
Merge pull request #16 from fibric/fibric-patch-1

new paragraph

Commit:
fixed typo and bad sub sentence

Commit:
Merge pull request #17 from fibric/fibric-patch-1

fixed typo and bad sub sentence

fibric commented:
@iojs/iojs-de 
I've added a new paragraph. Please read, check, comment and blame on me.
Nearly all of your comments are already taken into account.

nerdalytics committed:
Merge branch 'master' of https://github.com/iojs/website

# Conflicts:
#	content/de/es6.md

fibric commented:
@snostorm I have resolved the merge conflict

snostorm commented:
Great. I'll wait for the read/check you requested from a fellow @iojs/iojs-de member before merging.

fhemberger commented on content/de/es6.md:
```diff
@@ -1,5 +1,81 @@
-<!-- de i18n team, feel free to put a redirect notice here -->
+Die io.js-Entwicklung basiert auf der aktuellen Version der [V8](https://code.google.com/p/v8/)-Engine. Mit der Bestrebung, ständig die aktuellste Version der V8-Engine einzusetzen, stellen wir sicher, dass neue Funktionen der [JavaScript ECMA-262 Spezifikation](http://www.ecma-international.org/publications/standards/Ecma-262.htm) sowie Geschwindigkeits- und Stabilitätsverbesserungen schnell für ...
+
+Die Version {{project.current_version}} io.js wird mit der V8-Version {{project.current_v8}} ausgeliefert, welche bereits ES6-Funktionen beinhaltet, die deutlich weiterentwickelter sind, als jene der Version 3.28.73, die in joyent/node@0.12.x enthalten sind.
```
"Die Version {{project.current_version}} _von_ io.js"

Suggestion:
"die deutlich weiterentwickelter sind, als die der Version 3.28.73, welche in joyent/node@0.12.x enthalten ist."

fhemberger commented on content/de/es6.md:
```diff
@@ -1,5 +1,81 @@
+# ES6 und io.js
 
-<!-- de i18n team, feel free to put a redirect notice here -->
+Die io.js-Entwicklung basiert auf der aktuellen Version der [V8](https://code.google.com/p/v8/)-Engine. Mit der Bestrebung, ständig die aktuellste Version der V8-Engine einzusetzen, stellen wir sicher, dass neue Funktionen der [JavaScript ECMA-262 Spezifikation](http://www.ecma-international.org/publications/standards/Ecma-262.htm) sowie Geschwindigkeits- und Stabilitätsverbesserungen schnell für ...
```
Suggestion: "Mit dem Bestreben"

fhemberger commented on content/de/es6.md:
```diff
@@ -1,5 +1,81 @@
+
+## Überflüssiger Startparameter `--harmony`
+
+In joyent/node@0.12.x (V8 3.28+) aktiviert der Startparameter `--harmony` alle ES6-Funktionen auf einmal. Enthalten sind alle jene Funktionen, die als **completed** (Fertiggestellt), **staged** (Auslieferungsvorbereitung) und **in progress** (Entwicklungsphase) gekennzeichnet sind. (Mit der Ausnahme des _nonstandard/non-harmonious_ Ausdrucks `typeof`, welcher sich hinter dem Startparameter `--har ...
```
Please check the last sentence again, I think it should rather be:
"Daraus hätte sich die Best-Practice ergeben, entweder nur bestimmte Funktionen durch einzelne Startparameter verfügbar zu machen (z.B. `--harmony-generators`), oder einfach alle Funktionen einzuschalten, aber dann nur eine sehr restriktive Untermenge an Funktionen zu verwenden."

fhemberger commented on content/de/es6.md:
```diff
@@ -1,5 +1,81 @@
+
+Das Startparameter `--harmony` schaltet lediglich die **staged**-Funktionen ein und ist als Synonym zu `--es_staging` zu verstehen. Wie oben bereits beschrieben, werden hierdurch mehr oder weniger vollständige Funktionen aktiviert, die noch nicht als stabil zu bezeichnen sind. Wer auf Nummer sicher gehen will, gerade in produktiven Umgebungen, sollte ernsthaft darüber nachdenken, diesen Startpara ...
+
+## Wie erkenne ich, mit welcher V8 Version io.js ausgeliefert wurde?
```
V8-Version

fhemberger commented on content/de/es6.md:
```diff
@@ -1,5 +1,81 @@
+
+## Wie erkenne ich, mit welcher V8 Version io.js ausgeliefert wurde?
+
+io.js bietet eine einfache Möglichkeit, alle Abhängigkeiten der jeweilig installierten Version, durch das globale Objekt `process` aufzulisten. Durch folgenden Konsolen-Befehl kann die Version der V8-Engines ausgegeben werden:
```
V8-Engine

fibric replied:
Ich finde keine Stelle an der `V8 Engine` vorkommt um es durch `V8-Engine` ersetzen zu können.

fhemberger commented:
Just found a few minor things at first glance, the rest looks good to me for a merge.

Some sentences still sound a little bit off, because of with all that version numbers and technical stuff thrown in, but this is only a matter of style, which can be optimized later. The translation it self is sound and complete, @fibric did a great job with adding all the suggestions and corrections.

Commit:
Add suggested minor changes

Commit:
Merge pull request #18 from fibric/fibric-patch-1

Add suggested minor changes

fibric commented:
@fhemberger thanks. I've add your suggested changes.

fhemberger commented on content/de/es6.md:
```diff
@@ -1,5 +1,81 @@
+
+## Wie erkenne ich, mit welcher V8-Version io.js ausgeliefert wurde?
+
+io.js bietet eine einfache Möglichkeit, alle Abhängigkeiten der jeweilig installierten Version, durch das globale Objekt `process` aufzulisten. Durch folgenden Konsolen-Befehl kann die Version der V8-Engines ausgegeben werden:
```
"Durch folgenden Konsolen-Befehl kann die Version der V8-Engine~~s~~ ausgegeben werden"
-> V8-Engine

fibric replied:
ahh. das 's' :-)
ok wird geändert.

Commit:
Update es6.md

Commit:
Merge pull request #19 from fibric/fibric-patch-1

fix typo

snostorm commented:
Alright, I think we're good to merge this now? :)

snostorm commented:
Now live at https://iojs.org/de/es6.html -- thanks again @fibric @fhemberger

fibric commented:
@snostorm we have to thank you :)
I can delete my fork now?

snostorm commented:
Yup, go ahead
